### ISSUE TITLE
GameGuru MAX - Request - Name Storyboard HUD Screens

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#4357]
I think it's pretty important that we be able to name any screens we create in the Storyboard Editor. Take the new HUD screens, for initial consideration:

![huds](https://user-images.githubusercontent.com/36735507/228956111-e0939bc6-7ba2-495e-bcbf-3b3432285e86.png)

Other than the first being named "In-Game HUD", the rest are simply HUD Screen 2, 3, etc. At a glance, it could be difficult, especially for a new user, to see which is the Quest HUD and which is the Recipe HUD, etc. And since end-users are going to be able to create a variety of HUDs for use in their games, I think it is important that we be able to name them so we can find them quickly.

If there's currently a way to already name these HUD screens, I didn't see it. But I've also not had enough coffee today ... :P

**Comment by davetgc** [comment]
I agree this would be far more intuitive.

**Comment by Tum1370** [comment]
I agree with this, I remember asking Lee in one of the live streams about having the option to rename HUD screens.

**Comment by Necrym59** [comment]
Yes but lets get them working properly first

**Comment by MonkeyFrogStudio** [comment]
> Yes but lets get them working properly first

Of course. ;)
